### PR TITLE
add update-wiki workflow

### PR DESCRIPTION
**Pull request by rfomin**

### CODE REVIEW

File: .github/workflows/update-wiki.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          mv -f "./code/build-linux.md" "./wiki/Building-on-Linux.md"
+          mv -f "./code/build-windows.md" "./wiki/Building-on-Windows.md"
+          mv -f "./code/build-macos.md" "./wiki/Building-on-Mac.md"
+          mv -f "./code/cheats.md" "./wiki/New-Cheats.md"
+          mv -f "./code/controls.md" "./wiki/New-Controls.md"
+          mv -f "./code/crispness.md" "./wiki/Crispness-Menu.md"
+          mv -f "./code/params.md" "./wiki/New-Command-line-Parameters.md"
+          mv -f "./code/compatibility.md" "./wiki/Compatibility.md"
+          mv -f "./code/faq.md" "./wiki/FAQ.md"
+          mv -f "./code/changelog-history.md" "./wiki/Changelog History.md"
```
Comment (fabiangreffrath) [inline .github/workflows/update-wiki.yml:36]: Space in file name?

**Comment by rfomin** [comment]
I don't have permission to edit the wiki from PR. Is this approach OK? Then we can merge it, it will work from the master branch.

**Comment by fabiangreffrath** [comment]
Why do you use `mv` instead of `cp`?

File: .github/workflows/update-wiki.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+name: update wiki
+
+on:
+  push:
+    branches:
+      - "master"
+  pull_request:
```
Comment (fabiangreffrath) [inline .github/workflows/update-wiki.yml:7]: Does this mean that pull requests will already override our Wiki?

**Comment by rfomin** [comment]
> Why do you use `mv` instead of `cp`?

I never knew how to use these tools properly 😆 Please write a better command, I will gladly use it.

File: .github/workflows/update-wiki.yml (hunk 2 above)
Comment (rfomin) [reply .github/workflows/update-wiki.yml:7]: I will remove it. I wanted to try running it from PR.

**Comment by fabiangreffrath** [comment]
> I never knew how to use these tools properly 😆 Please write a better command, I will gladly use it.

I'd use `cp -af` instead of `mv -f`.

**Comment by kiwphi** [comment]
Seems like relative links (e.g ./wiki/compatiblity) prepends /blob/master/ to all links, i'll have to fix these in the readme when you're done

**Review by fabiangreffrath — APPROVED** [review]

**Pull request merged**